### ISSUE TITLE
Automatic wheels build

### ISSUE THREAD

**Issue by biphasic** [neuromorphicsystems/aedat#12]
Hello Alex,
As a maintainer of another package that would like to rely on an .aedat decoder, it is preferable if that package is lightweight, fast and easy to download/install. `aedat` is fast and lightweight, but is only easy to install as long as there are wheels available.

Currently, you are the only one who builds wheels for new system/Python configurations. I saw that you even wrote a separate library to help do this. I was wondering, do you think it would be possible to automate this process using Github actions for example? I found this Action: https://github.com/pypa/cibuildwheel which seems to aim at that. 

It would make your life easier and guarantee up-to-date binaries for users. Would be great to get your opinion on that!

Gregor

**Comment by aMarcireau** [comment]
Hey Gregor,

I absolutely agree with you, and Github actions + cibuildwheel is an excellent solution! Github actions were not quite ready (in particular their macOS machines) when I started the project, which is why I created my own separate library. I am planning to switch to Github actions relatively soon.

I'll try to get to it by the end of the year, but I can move the taask up my to-do list if you have a deadline and need aedat to be stable sooner.

Cheers,
Alex
